app/javascript/onboarding/components/ClosingSlide.jsx

```diff
@@ -1,22 +1,35 @@
 import { h, Component } from 'preact';
 import PropTypes from 'prop-types';
 
-import { getContentOfToken } from '../utilities';
+import { updateOnboarding } from '../utilities';
 
 class ClosingSlide extends Component {
   componentDidMount() {
-    const csrfToken = getContentOfToken('csrf-token');
-    fetch('/onboarding_update', {
-      method: 'PATCH',
-      headers: {
-        'X-CSRF-Token': csrfToken,
-        'Content-Type': 'application/json',
-      },
-      body: JSON.stringify({ user: { last_onboarding_page: 'closing slide' } }),
-      credentials: 'same-origin',
-    });
+    updateOnboarding('closing slide');
   }
 
+  whatsNext() {
+    const properties = [
+      { href: '/welcome', text: 'Join the Welcome Thread', emoji: '😊' },
+      { href: '/new', text: 'Write your first DEV post', emoji: '✍️' },
+      { href: '/top/infinity', text: 'Read all-time top posts', emoji: '🤓' },
+      { href: '/settings', text: 'Customize your profile', emoji: '💅' },
+    ];
+
+    return(
+      properties.map((element, index) => 
+          <a key={ index } href={ element.href } data-no-instant={(index === 0) ? true : null }>
+            { element.text }
+            <p className="whatnext-emoji">
+              <span role="img" aria-label="tada">
+                { element.emoji }
+              </span>
+            </p>
+          </a>   
+      )
+    );
```

Comment:
Doesn't add much to have a method return a small piece of component here.
As a suggestion:

1. make `whatsNext()` method return just the array of objects.
2. `.map` it in `render()` method